[ADD] sms marketing: added essentials/getting started docs (#2553)
odoo/documentation: content/applications/marketing/sms_marketing/essentials/sms_essentials.rst

customers, etc.) will receive the SMS, unless a specific recipient rule(s) are entered.

In the example below, the message will only be sent to contacts who haven't chosen to
:guilabel:`Blacklist` their contact information through the :guilabel:`Unsubscribing` process.

.. image:: sms_essentials/contact-recipient.png
   :align: center
   :alt: Contact recipients on SMS marketing.

How to create SMS messages
==========================

Enter the content of the SMS in the text field, found in the :guilabel:`SMS Content` tab. Links and
emojis can also be included. Beneath the text field, Odoo displays how many characters are used in
the message, along with how many SMS mailings it will take to deliver the complete message.

.. tip::
   To check the price of sending an SMS for a country, click on the :guilabel:`Information` icon.

.. image:: sms_essentials/sms-price-check.png
   :align: center
   :alt: SMS price check icon

.. note::
   Credits must be purchased from Odoo in order to take advantage of the app (and its features).
   SMS messages will not be sent without credits.

.. seealso::
   - :doc:`../pricing/pricing_and_faq`

> How to track links used in SMS messages
> ---------------------------------------

Reviewer: Note for all of the headings that start with `How to...`
We can just remove that part and stick with the action that the heading conveys for the section. Please apply to all headings in this and other docs.

```suggestion
Track links used in SMS messages
--------------------------------
```

When links are used, Odoo automatically generates link trackers to gather analytical data and
metrics related to those specific links, which can be found by going to
:menuselection:`Configuration --> Link Tracker`.

.. image:: sms_essentials/sms-link-tracker.png
   :align: center
   :alt: SMS Link Tracker page.

How to adjust SMS Settings
==========================

Under the :guilabel:`Settings` tab of the SMS template, there is an option to :guilabel:`Include
opt-out link`. If activated, the recipient is able to unsubscribe from the mailing list, thus
avoiding all future mailings.

An employee can be designated as the :guilabel:`Responsible` in the :guilabel:`Tracking` section of
the :guilabel:`Settings` tab, as well.

> .. image:: sms_essentials/sms-settings-tab.png

Reviewer: Too wide. Doesn't appear to be in HD.

   :align: center
   :alt: SMS Settings tab.

How to send SMS messages
========================

Once a mailing is created, choose when Odoo should deliver the message from the following options:

> - :guilabel:`Send`: sends the message immediately. Consider using this option if the recipient list
>   isn't excessive.

Reviewer: Point of clarity: what does `excessive` mean here? How is this defined?

Consider the following suggestion, instead:
```suggestion
- :guilabel:`Send`: sends the message immediately. Consider using this option if the recipient list
  is highly refined or in cases that involve fast approaching deadlines, such as a flash sale.
```

- :guilabel:`Schedule`: choose a day (and time) for Odoo to send the mailing. This is typically the
  best option for mailings related to a specific event. Such a method can also be used to promote a
  limited-time offer, or to help plan a company's content strategy in advance.
- :guilabel:`Test`: allows for an SMS to be sent to one or multiple numbers for test purposes.
  Remember to use a comma between phone numbers, if multiple numbers are used as recipients.

Visualize reports
=================

> On the :guilabel:`Reporting` page (accessible via the :guilabel:`Reporting` option in the header
> menu), there are options to apply filters, measures, and view metrics in a number of different
> layouts (:guilabel:`Graph`, :guilabel:`List`, and :guilabel:`Cohort`).

Reviewer: I think in this case, even though it's solitary, the second tag is actually a `menuselection`. As well, I would indicate`filters` and `measures` as `guilabels`. Can remove the parenthesis in the last sentence too and just write out what those views are.
```suggestion
On the :guilabel:`Reporting` page (accessible via the :menuselection:`Reporting` option in the
header menu), there are options to apply different combinations of :guilabel:`Filters` and
:guilabel:`Measures` to view metrics in a number of different layouts, as in the :guilabel:`Graph`,
:guilabel:`List`, and :guilabel:`Cohort` views.
```

Each :guilabel:`Reporting` metric view option allows for more extensive performance analysis of SMS
mailings.

For example, while in the default :guilabel:`Graph` view, SMS data is visualized as different graphs
and charts, which can be sorted and grouped in various ways (e.g. :guilabel:`Measures` drop down
menu).

> .. image:: sms_essentials/sms-reporting-page.png

Reviewer: The crop is pretty good here, however this image is too wide @ 1920 pixels. Consider taking a shot that has more data, as well, instead of what we get by default using runbot. Images that show the depth of capabilities, I think, are more valuable to the reader and justifies adding the extra file weight to the page.
